### PR TITLE
Restore local server polling capabilities

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: db-client/src/db.ts [hunk 1]
```diff
@@ -81,19 +83,40 @@ export class DB {
...
+  /**
+   * Polls on updates to specified keys since specified versions.
+   * @see KeyedVersions
+   * @see KeyedPatches
+   */
+  public async poll(
+    keysToVersions: Array<[string, Version]>,
+    opts: PollOptions = {},
+  ): Promise<Array<[string, Patch[]]>> {
+    return this.client.poll(this.ctx, keysToVersions, opts);
```
Comment (arielshaqed) [inline db-client/src/db.ts]: Nit: `async` _and_ not `await`?

File: db-testsuite/src/index.ts [hunk 2]
```diff
@@ -185,7 +203,120 @@ test('DB.update does nothing if document not updated', async (t) => {
...
+  const { client } = t.context;
+  await client.create('test1', 'a');
+  const { version } = (await client.startPolling('test1'))!;
+  const [keyedPatches] = await Promise.all([
+    client.poll([['test1', version]]),
+    client.remove('test1'),
+  ]);
+  t.deepEqual(keyedPatches, [
+    ['test1', [
+      { version: { major: version.major, minor: 2 }, ops: [{ op: 'remove', path: '/root' }] },
```
Comment (arielshaqed) [inline db-testsuite/src/index.ts]: Isn't this `incrVersion(version)`?

File: db-testsuite/src/index.ts [hunk 3]
```diff
@@ -219,3 +350,13 @@ test('DB.update throws TypeError if trying to modify returned object', async (t)
     return obj;
   }), TypeError);
 });
+
+test('DB.update sets operartionId', requiresPolling, async (t) => {
```
Comment (arielshaqed) [inline db-testsuite/src/index.ts]:
```suggestion
test('DB.update sets operationId', requiresPolling, async (t) => {
```

File: db-client/src/db.ts (hunk 1 above)
Comment (bergundy) [reply db-client/src/db.ts]: I agree with @vogre that this way is cleaner.

File: db-testsuite/src/index.ts (hunk 2 above)
Comment (bergundy) [reply db-testsuite/src/index.ts]: Yes, thanks

File: db-testsuite/src/index.ts (hunk 3 above)
Comment (bergundy) [reply db-testsuite/src/index.ts]: Thanks!

**Pull request merged**